src/engine/cache_engine.c

```diff
@@ -1,5 +1,5 @@
 /*
```

Comment:
Nope. It's also used in `ocf_get_io_iface_name()`.

Reply:
Ok. I still feel there are two functions doing almost the same thing. Maybe ocf_cache_mode_to_engine_cb() should call ocf_get_io_if() to avoid this?

Reply:
Taking into account that `ocf_get_io_if()` is not static (and so it will not be inlined) it doesn't seem right to call it just for sake of additional justification for it's existence. I think I'll just merge it into `ocf_get_io_iface_name()`.